ocaml/xapi/cert_distrib.mli

```diff
@@ -6,6 +6,12 @@ val exchange_certificates_among_all_members : __context:Context.t -> unit
     certificates from all members in a pool and installed on all of them. On
     success, new bundles will have been generated on the members. *)
 
+val am_i_missing_certs : __context:Context.t -> bool
```

Comment:
Would it be more useful to return the list or set of missing certs?

Reply:
We could do that. At the moment the logic is roughly: if i am missing any certs then ask the primary host to send me all its certs

If we were to request a subset of certs then i guess this would be more efficient on the network, but with a possible increased risk of certs becoming stale (if we consider the primary host as the source of truth). The 'am i missing certs thread' should only very rarely actually copy certs from one host to another, so I doubt it would be a problem